core/src/main/java/org/stellar/anchor/util/SepHelper.java

```diff
@@ -70,6 +71,20 @@ public static void validateAmount(String messagePrefix, String amount) throws An
     if (sAmount.signum() < 1) {
       throw new BadRequestException(messagePrefix + "amount should be positive");
     }
+    return sAmount;
+  }
+
+  public static void validateAmountLimit(String messagePrefix, String amount, Long min, Long max)
+      throws AnchorException {
+    BigDecimal sAmount = validateAmount("", amount);
+    BigDecimal bdMin = new BigDecimal(min);
+    BigDecimal bdMax = new BigDecimal(max);
+    if (sAmount.compareTo(bdMin) == -1) {
+      throw new BadRequestException(messagePrefix + "amount less than min limit");
+    }
+    if (sAmount.compareTo(bdMax) == 1) {
+      throw new BadRequestException(messagePrefix + "amount exceeds max limit");
```

Comment:
💡 We may consider using `String.format()` to format a Java String.

Reply:
Changed this to use String.format